File: Retrofit/app/src/main/java/com/example/arnassmicius/retrofit/api/service/GitHubClient.java
package com.example.arnassmicius.retrofit.api.service;

import com.example.arnassmicius.retrofit.api.model.GitHubRepo;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

/**
 * Created by arnas on 18.2.16.
 */

public interface GitHubClient {

    @GET("/users/{user}/repos")
    Call<List<GitHubRepo>> reposForUser(@Path("user") String user);
}

File: Retrofit/app/src/main/java/com/example/arnassmicius/retrofit/api/service/GitHubClient.kt
package com.example.arnassmicius.retrofit.api.service

import com.example.arnassmicius.retrofit.api.model.GitHubRepo

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

/**
 * Created by arnas on 18.2.16.
 */

interface GitHubClient {

    @GET("/users/{user}/repos")
    fun reposForUser(@Path("user") user: String): Call<List<GitHubRepo>>
}
